gremlin-go/driver/bytecode.go

```diff
@@ -22,4 +22,5 @@ package gremlingo
 type bytecode interface {
 	addSource(sourceName string, args ...interface{})
 	addStep(stepName string, args ...interface{})
+	toString() string
```

Comment:
Didn't I remove this in my PR?

This probably will change when you rebase.

Reply:
Yes, that was still before the rebase.